### PR TITLE
chore(rn): add release note of 2021.1-0617

### PR DESCRIPTION
**Pull request by baptistemesta**

### CODE REVIEW

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+=== Fixes in Bonita 2021.1-0617 (2021-06-17)
+
+* UID-440 - dataTable don't sort date related columns when source is "Variable"
+* UID-439 - Issue exporting and importing a custom widget with assets
+* UID-431 - Layout title metadata displayed and persisted instead of the living app display name when the living application link is shared
+* UID-425 - Japanese locale correspondence table is wrong
+* UID-362 - Import custom widgets doesn't import assets
+* UID-298 - Error when entering a description for a fragment
+* STUDIO-3979 - multiInstantiator variable cannot use the Java setter operator in operation
+* STUDIO-3965 - Regression: Organization is reset to Acme or None after restart
```
Comment (rbioteau) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* STUDIO-3965 - Active organization preference is not stored
```

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* STUDIO-3953 - LA Builder is slow
```
Comment (rbioteau) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* STUDIO-3953 - Improve LA-Builder build performence
```

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* STUDIO-3948 - Git clone should not display migration warning message
```
Comment (rbioteau) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* STUDIO-3948 - Git clone should not display migration warning message when not causing breaking changes
```

File: modules/ROOT/pages/release-notes.adoc [hunk 4]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
 
 == Bug fixes
 
+=== Fixes in Bonita 2021.1-0617 (2021-06-17)
+
+* UID-440 - dataTable don't sort date related columns when source is "Variable"
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* UID-440 - dataTable does not sort date-related columns when source is "Variable"
```

File: modules/ROOT/pages/release-notes.adoc [hunk 5]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* UID-439 - Issue exporting and importing a custom widget with assets
+* UID-431 - Layout title metadata displayed and persisted instead of the living app display name when the living application link is shared
+* UID-425 - Japanese locale correspondence table is wrong
+* UID-362 - Import custom widgets doesn't import assets
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* UID-362 - Import custom widgets does not import assets
```

File: modules/ROOT/pages/release-notes.adoc [hunk 6]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
 
 == Bug fixes
 
+=== Fixes in Bonita 2021.1-0617 (2021-06-17)
+
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:236]: ==== Fixes in Bonita Development Suite (Studio and UI Designer)

File: modules/ROOT/pages/release-notes.adoc [hunk 7]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* UID-440 - dataTable don't sort date related columns when source is "Variable"
+* UID-439 - Issue exporting and importing a custom widget with assets
+* UID-431 - Layout title metadata displayed and persisted instead of the living app display name when the living application link is shared
+* UID-425 - Japanese locale correspondence table is wrong
+* UID-362 - Import custom widgets doesn't import assets
+* UID-298 - Error when entering a description for a fragment
+* STUDIO-3979 - multiInstantiator variable cannot use the Java setter operator in operation
+* STUDIO-3965 - Active organization preference is not stored
+* STUDIO-3953 - Improve LA-Builder build performence
+* STUDIO-3948 - Git clone should not display migration warning message when not causing breaking changes
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:248]: ==== Fixes in Bonita Runtime (including Portal)

File: modules/ROOT/pages/release-notes.adoc [hunk 8]
```diff
@@ -232,6 +232,25 @@ Use BPMN 2 importer for model exchange with third-party editors.
...
+* RUNTIME-297 - start-bonita.sh can't be launched from outside of bundle directory
+* RUNTIME-293 - Session fixation protection breaks some SAML features
+* RUNTIME-212 - Process manager typo into dynamic-permissions-checks(-custom).properties file
+* RUNTIME-33 - Impossible to install a valid BDM after an attempts to install an invalid BDM
+Refresh results
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc]: ?

**Review by NathalieC — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/release-notes.adoc (hunk 8 above)
Comment (baptistemesta) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
```

**Review by NathalieC — APPROVED** [review]

**Pull request merged**